crates/server/src/models/service.rs

```diff
@@ -41,6 +41,20 @@ pub struct ServiceForCreate {
     pub retry_interval: u16,
 }
 
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct ServiceForUpdate {
```

Comment:
This structure supports partial updates which makes it useful for flexibility in service management.